code/string_algorithms/src/suffix_array/suffix_array.c

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
+#include <cstring>
+#include <algorithm>
+using namespace std;
```

Comment:
C or C++? Pick a language? Currently, this file makes no sense at all.

Reply:
also don't use "using namespace std" you should std::cout(or whatever) .

Reply:
@dragon540 That depends on whether this is C or C++...